tools/CheckDates.scala

```diff
@@ -0,0 +1,184 @@
+import sys.process._
+
+object CheckDates {
```

Comment:
`CopyrightDateTool` is an improved name because,

- It is more specific
- It requires no explanation as to it's purpose